airbyte-webapp/src/locales/en.json

```diff
@@ -269,6 +269,7 @@
   "sources.cancelled": "Cancelled",
   "sources.incomplete": "Retrying",
   "sources.partialSuccess": "Partial Success",
+  "sources.streamsReset": "{count, plural, =0 {0 streams} one {# stream} other {# streams}} reset",
```

Comment:
@andyjih The design currenlty states "x streams reset". Should that rather be "x streams resetted"?

Reply:
@timroes "reset" is also the past tense for the word. 

None: "No streams were reset"
One: "1 stream was reset"
Many: "N streams were reset".

Reply:
#TIL :D